scripts/release/README

```diff
@@ -0,0 +1,20 @@
+This document describes how to make an ADAM release.
```

Comment:
Ha, sorry. I copied these scripts from another project that I work on and forgot to change the documentation file. My apologies.